django/db/migrations/operations/base.py

```diff
@@ -85,6 +98,16 @@ def describe(self):
         """
         return "%s: %s" % (self.__class__.__name__, self._constructor_args)
 
+    def formatted_description(self):
+        """
+        Output the describe output in a more informative way.
+        """
+        description = self.describe()
+        category = (
+            "{} ".format(self.category.value) if self.category is not None else ""
+        )
+        return "{}{}".format(category, description)
+
```

Comment:
Late to the party here but is there a reason to have both `describe` and `formatted_description`? Any reason not to alter `describe`?

Otherwise this appears to be a good candidate for the usage of `f` strings.

Reply:
Also late just back from summer break…

Agree with Simon.

Would this be more readable?

```python
def formatted_description(self):
    if self.category is None:
        return self.describe()
    return f"{self.category.value} {self.describe()}"
```

Reply:
> Late to the party here but is there a reason to have both `describe` and `formatted_description`? Any reason not to alter `describe`?

We have defined `describe()`  in all operations so I would not change them.

> Otherwise this appears to be a good candidate for the usage of `f` strings.

Agreed :+1:

Reply:
> Also late just back from summer break…
> 
> Agree with Simon.
> 
> Would this be more readable?
> 
> ```python
> def formatted_description(self):
>     if self.category is None:
>         return self.describe()
>     return f"{self.category.value} {self.describe()}"
> ```

We don't all function calls in f-strings.